### ISSUE TITLE
Namespaces are placed in a new line

### ISSUE THREAD

**Issue by davity** [DotJoshJohnson/vscode-xml#13]
When formatting a document, every namespace attribute is placed on a new line.

Example:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<parent xmlns="namespace.url" xmlns:foo="bar">
   <child>Info</child>
</parent>
```

is converted to

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<parent 
   xmlns="namespace.url" 
   xmlns:foo="bar">
   <child>Info</child>
</parent>
```

It will be nice to have at least an option to disable this behaviour and put every node tag on a unique line with all its attributes.

**Comment by DotJoshJohnson** [comment]
Fixed in 08c81f7.

**Comment by DotJoshJohnson** [comment]
Fixed in [version 1.1.0](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v1.1.0) (now on marketplace).

A new setting has been created, `xmlTools.splitXmlnsOnFormat`. Setting this to `false` will keep xmlns attributes on the same line when formatting XML.

**Comment by davity** [comment]
Wow, that was fast, thank you!

Now there's a related issue. If a document were formatted with `xmlTools.splitXmlnsOnFormat` set to `true` (default behaviour) and then formatted again with option changed to `false`, document won't change: `xmlns` tags will remain on their own lines.
On the other side, if formatted with split option at `false` and then again with it at `true`, `xmlns` tags would be separated on new lines.

**Comment by DotJoshJohnson** [comment]
No problem!

This is technically an issue with the minification function as XML is first linearized before it is formatted. The minifier is leaving newlines in place as it is only removing them from between `>` and `<`. I've created issue #16 to track this correction.

**Comment by davity** [comment]
Ok, thank you for your support!

**Comment by DotJoshJohnson** [comment]
Fixed in [version 1.3.1](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v1.3.1) (now on marketplace).
